metadetect/fitting.py

```diff
@@ -317,12 +317,15 @@ def fit_all_psfs(mbobs, psf_conf, rng):
             psf_obs = obs.get_psf()
             fit_one_psf(psf_obs, psf_conf, rng)
 
+
 def fit_one_psf(obs, pconf, rng):
-    Tguess=4.0*obs.jacobian.get_scale()**2
+    # Tguess=4.0*obs.jacobian.get_scale()**2
+    fwhm_guess = 0.9
```

Comment:
This is probably fine, but potentially DES-specific.

Reply:
It is, but no more so than the one it replaced

Reply:
this one is just easier to think about, since psf fwhm is a number everyone knows for DES